lib/shipit/line_buffer.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Shipit
+  class LineBuffer
+    SEPARATOR = "\n"
+
+    def initialize(value = "")
+      @value = value
+    end
+
+    def buffer(text, &block)
+      @value += text
```

Comment:
Might not matter in the grand scheme of things, but mutating the string with `<<` is likely much more performant that constant concatenation.

Reply:
Good catch! Yes, I used `+=` because `@value` was a frozen string literal by default. Changing that by duplicating it in the initialiser.